src/carnot/core/data/auto_retrieval/quest_data_prep.py

```diff
@@ -0,0 +1,257 @@
+from __future__ import annotations
+
+import os
+import re
+import json
+import hashlib
+import logging
+import unicodedata
+from dataclasses import dataclass
+from typing import List, Dict, Any, Iterable, Optional
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_TOKENIZER_MODEL = "BAAI/bge-small-en-v1.5"
+
+
+# -----------------------------
```

Comment:
Is this copied from the baseline experiments? That probably shouldn't live within the auto retrieval package.